### DISCUSSION TITLE
Auto Rejoin

### DISCUSSION THREAD

**Discussion by Castro3halo** [runelite/runelite#15397] [Q&A] [answered]
Feature to automatically rejoin prior party when logging back in, instead of having to manually rejoin each time I login to PVM with the same group.

**Comment by MESLewis** [comment, marked as answer]
This is not being considered because it will increase the load on the party server.

**Comment by Drklich256** [comment]
Search menue for works so u can find a specific world faster

**Comment by Drklich256** [comment]
I ment world's
